disperser/cmd/dataapi/flags/flags.go

```diff
@@ -146,6 +146,13 @@ var (
 		Value:    6 * time.Minute,
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "TRANSACTION_TIMEOUT"),
 	}
+	NonsigningRateThresholdFlag = cli.IntFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "nonsigning-rate-threshold"),
+		Usage:    "only operators with nonsigning rate >= this threshold are eligible for ejection, this value must be in range [10, 100], any value not in this range means disabling this flag",
```

Comment:
disabling the flag means, eject based on regular SLA?

Reply:
Yes, no additional conditions to affect the standard SLA and ejection flow.